docs/guides/cloud/github-integration.mdx

```diff
@@ -137,11 +137,6 @@ to GitHub.com to complete the installation:
 2. Choose which repositories to associate with the Cypress GitHub App. You can
    choose **All repositories** or **Only select repositories**. with your
    Cypress GitHub App installation.
-   <DocsImage
-     src="/img/dashboard/github-integration/select-gh-repos.jpg"
-     alt="Select GitHub repositories"
-     width={600}
-   />
 
 :::info
 
```

Comment:
This info block can use markdown syntax for consistency with the rest of the doc

```suggestion
:::info

All current and _future_ repositories will be included with this installation if
you choose **All repositories**.

:::
```